tools.js

```diff
@@ -589,23 +587,6 @@ module.exports = (() => {
 
 					banlistTable[subformat.banlist[i]] = subformat.name || true;
 					banlistTable[toId(subformat.banlist[i])] = subformat.name || true;
-
-					let complexList;
-					if (subformat.banlist[i].includes('+')) {
-						if (subformat.banlist[i].includes('++')) {
-							complexList = subformat.banlist[i].split('++');
-							for (let j = 0; j < complexList.length; j++) {
-								complexList[j] = toId(complexList[j]);
```

Comment:
Because the original intent was code simplification; the correction of the message turned out to be a bonus. See whether you like my other approach.